website/docs/reference/node-selection/state-comparison-caveats.md

```diff
@@ -60,7 +60,7 @@ models:
 
 </File>
 
-Of course, if the raw Jinja expression is modified, it will be marked as a modification.
+Of course, if the raw Jinja expression is modified, it will be marked as a modification unless you set the `state_modified_compare_more_unrendered` behavior flag to `True`. Learn more about [Behavior flags](/reference/global-configs/behavior-changes#behavior-change-flags).
```

Comment:
@graciegoheen or @jtcohen6 Does this need to be versioned? It's unclear in [the Core PR](https://github.com/dbt-labs/dbt-core/pull/10487/files) whether this was backported or not?

Reply:
I do not believe this was backported - let's version this as 1.9+

Reply:
I don't think this is quite the right spot for the callout. 
<img width="886" alt="Screenshot 2024-10-01 at 10 52 52 PM" src="https://github.com/user-attachments/assets/83829421-a82b-4a94-ab60-e1b294eb396c">

maybe we add it to the second bullet? "Env-aware logic that causes different behavior based on the target, env vars, etc." and then have separate docs in this section for 1.9?

Reply:
Applied this change in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/6207/commits/b3c93a148ec2b3124696e04ae0dfc7b8edd4a825)!